mappings/net/minecraft/packet/play/RideEntityS2C.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_308 net/minecraft/packet/play/RideEntityS2C
```

Comment:
StartRidingS2C was fine don't need to change it smh

Reply:
ok